scout/bin.py

```diff
@@ -22,13 +22,13 @@ class SolvParser(object):
 
     def __init__(self):
         self.pool = solv.Pool()
-        self.parser = SafeConfigParser()
 
         for repofile in [ f for f in os.listdir(self.etcpath) if fnmatch(f, '*.repo') ]:
             try:
-                name = os.path.splitext(repofile)[0]
-                self.parser.read( '%s/%s' % (self.etcpath, repofile) )
-                if self.parser.get(name, 'enabled') == '1':
+                parser = SafeConfigParser()
+                parser.read( '%s/%s' % (self.etcpath, repofile) )
+                name = parser.sections()[0]
```

Comment:
This only reads the first section of the file.
A `.repo` file may contain several sections (i.e. several repositories) so you should loop on all the sections.

See also [comment 11](https://bugzilla.opensuse.org/show_bug.cgi?id=980393#c11) on [boo#980393](https://bugzilla.opensuse.org/show_bug.cgi?id=980393).